rotkehlchen/tests/unit/decoders/test_omnibridge.py

```diff
@@ -0,0 +1,247 @@
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount)),
+            location_label=user_address,
+            notes=f'Burned {gas_amount} ETH for gas',
+            tx_hash=tx_hash,
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            sequence_index=415,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=A_USDT,
+            balance=Balance(amount=FVal(bridge_amount)),
+            location_label=user_address,
+            notes=f'Bridge {bridge_amount} USDT from Gnosis to Ethereum via Gnosis Chain bridge',
+            tx_hash=tx_hash,
+            counterparty=CPT_GNOSIS_CHAIN,
+            address=ETHEREUM_BRIDGE_ADDRESS,
+        ),
+    ]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x8D1D236F45a7DEcB769D88963b0F699cDBE5D833']])
+def test_omnibridge_ethereum_eth_withdrawal(database, ethereum_inquirer, ethereum_accounts):
+    tx_hash = deserialize_evm_tx_hash('0x0d5b64cb8b79e4a728b22caf7ac51a0a595278fd6b2265775a4429a204e4e5e3')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    user_address, timestamp, gas_amount, bridge_amount = ethereum_accounts[0], TimestampMS(1723786739000), '0.000333774965537772', '24.42848'  # noqa: E501
+    assert events == [
+        EvmEvent(
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount)),
+            location_label=user_address,
+            notes=f'Burned {gas_amount} ETH for gas',
+            tx_hash=tx_hash,
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(bridge_amount)),
+            location_label=user_address,
+            notes=f'Bridge {bridge_amount} WETH from Gnosis to Ethereum via Gnosis Chain bridge',
+            tx_hash=tx_hash,
+            counterparty=CPT_GNOSIS_CHAIN,
+            address=string_to_evm_address('0xa6439Ca0FCbA1d0F80df0bE6A17220feD9c9038a'),
+        ),
+    ]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('gnosis_accounts', [['0xcc2c6D82e00E14f81FFc8e4F6A26c4522adA5a34']])
+def test_omnibridge_gnosis_token_withdrawal(database, gnosis_inquirer, gnosis_accounts):
```

Comment:
what about gnosis -> ethereum xdai? And ethereum->gnosis xdai? All on gnosis. That's already covered in other tests?